docs/wiki/framework/events-framework.md

```diff
@@ -81,6 +81,7 @@ MenuType: 0 = Interaction, 1 = Self Interaction
 
 | Event Key | Parameters | Locality | Type | Description |
 |----------|---------|---------|---------|---------|---------|
+|`ace_allowDefuse` | [_mine, _allow] | Global | Callable | Set allowment of the dynamic defusal action on a mine
```

Comment:
"Allowance" maybe?

Reply:
Sound good. finding the right words is hard :)